cli/args/flags.rs

```diff
@@ -460,12 +460,16 @@ pub struct Flags {
   pub reload: bool,
   pub seed: Option<u64>,
   pub strace_ops: Option<Vec<String>>,
+
+  // NOTE(bartlomieju): keep these `unstable_*` in sync with
+  // `cli/standalone/binary.rs` otherwise some unstable features might not be
+  // enabled in standalone binaries.
```

Comment:
Perhaps these should be extracted out to a common struct and shared between the two?

Reply:
Done!